dask/dataframe/io/tests/test_parquet.py

```diff
@@ -125,14 +125,14 @@ def test_roundtrip_from_pandas():
 
 def test_categorical():
     with tmpdir() as tmp:
-        df = pd.DataFrame({'x': ['a', 'b', 'c'] * 100},
+        df = pd.DataFrame({'x': [u'a', u'b', u'c'] * 100},
```

Comment:
Don't we want that column name to be text rather than bytes anyway though?  I think it's standard to use `b'...'` when one wants bytes specifically.  The unicode_literals import just makes Python 2 adhere to Pyhton 3 standards, which we've been operating under for some time.  I would be entirely comfortable adding the unicode_literals to all .py files in the dask/dask repository.